webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTableConstants.js

```diff
@@ -1 +1,2 @@
 export const MODELS_TABLE_CONTROLLER = 'models';
+export const MODELS_TABLE_ID = 'models_table';
```

Comment:
nit: would it be possible to use camel case?

Reply:
I think it will make the redux logs hard to read because they use `tableID.toUpperCase()` as type for the table actions, so it will become MODELTABLE_REQUEST instead of MODEL_TABLE_REQUEST

Reply:
You are right, I haven't considered that. I was asking because it gives you `state.models_table` and I'd rather have one naming scheme:

![store-models](https://user-images.githubusercontent.com/2664090/62684266-98cde300-b9c0-11e9-9518-82207fb0347f.png)

I am also wondering why linter did not flag it in ModelsTable/index.js...